Formula/raxml-ng.rb

```diff
@@ -2,8 +2,8 @@ class RaxmlNg < Formula
   desc "RAxML Next Generation: faster, easier-to-use and more flexible"
   homepage "https://sco.h-its.org/exelixis/web/software/raxml/"
   url "https://github.com/amkozlov/raxml-ng.git",
-    :tag => "0.5.1",
-    :revision => "8a3d6af12fbff60239744595631a468275d5a02a"
+    :tag      => "0.7.0",
```

Comment:
```suggestion
    :tag => "0.7.0",
```
Please remove the additional white space.

Reply:
Audit demands this:

```
brewsci/bio/raxml-ng:
  * C: 5: col 5: Align the elements of a hash literal if they span more than one line.
```